.env-example

```diff
@@ -1,6 +1,10 @@
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
 
+# Used for Google Classroom Roster Integration
+GOOGLE_CLIENT_ID=
```

Comment:
Can we add something to the README about this in the section about filling out the .env? Mainly is this optional? I have no idea how I'd go about getting a client ID and secret.

Reply:
Is it possible to add this later? This google integration PR is not quite ready as I need to polish it a bit more (duplicate roster entries and allow teachers to import different identifiers).

I'll also make a separate PR for this. This PR is quite large and I'm getting lost in the files changed already 😂